injector/src/main/java/edu/ucr/cs/riple/injector/Injector.java

```diff
@@ -147,14 +148,20 @@ public Set<FileOffsetStore> removeAnnotations(Set<RemoveAnnotation> requests) {
    */
   @Nullable
   public static CompilationUnit parse(Path path) {
-    // Set parser configuration to Java 17.
+    // TODO: Figure out how to differentiate a generated file from a non-generated file.
+    if (path == null) {
+      return null;
+    }
     ParserConfiguration parserConfiguration = new ParserConfiguration();
+    // Set parser configuration to Java 17.
     parserConfiguration.setLanguageLevel(ParserConfiguration.LanguageLevel.JAVA_17);
     StaticJavaParser.setConfiguration(parserConfiguration);
     try {
       return StaticJavaParser.parse(path);
     } catch (FileNotFoundException e) {
       return null;
+    } catch (NoSuchFileException e) {
```

Comment:
Let's not use both `NoSuchFileException` and `FileNotFoundException` since they can be collapsed into one, and use only `NoSuchFileException` from `java.nio.file` package and remove the other one.